src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -280,7 +280,7 @@ def dcos_install_cli(cmd, client, install_location=None, client_version='1.8'):
         raise CLIError('Connection error while attempting to download client ({})'.format(err))
 
 
-def k8s_install_cli(cmd, client, client_version='latest', install_location=None):
+def k8s_install_cli(cmd, client=None, client_version='latest', install_location=None):
```

Comment:
Then you can remove `client` from here.
Since the client_factory is None, the CLI won't call this method with `client` so you should be good.